### DISCUSSION TITLE
Image add glow

### DISCUSSION THREAD

**Discussion by Rhinemann** [ImageMagick/ImageMagick#7509] [Help] [answered]
### ImageMagick version

7.1.1-35

### Operating system, version and so on

OS: Kubuntu 23.10

### Description

I have used this: `magick in/CorGold.png -gaussian-blur 0x10 PNG:- | magick "$f" - -gravity center -composite -channel RGB +level-colors gold, out/CorGold.png` command to turn this image:
![CorGold](https://github.com/user-attachments/assets/d0d5cd39-31a5-4f9b-9ef6-49c733adafa1)
into this:
![CorGold](https://github.com/user-attachments/assets/2614e402-0920-4d6c-bd8c-82462573f0fa)

Is it possible to do the same in one command without piping?

**Comment by fmw42** [comment]
What is $f?  You do not show a second image!

**Reply by Rhinemann** [reply]
$f is a filename, I copied it from a larges script, and didn't notice, so it's in/CorGold.png too, in this case.
And the second image is showing on my side.
![image](https://github.com/user-attachments/assets/23a2e408-f5d5-4556-8807-d8c358615d59)

**Comment by snibgo** [comment]
As fmw42 says. But we can guess that substituting the "-" by the magick options in the first magick, within parantheses, might work:
```
magick "$f" ( in/CorGold.png -gaussian-blur 0x10 ) -gravity center -composite -channel RGB +level-colors gold, out/CorGold.png
```

**Reply by fmw42** [reply, marked as answer]
It looks like you might be on Unix (Kubuntu rather than Windows, if so put escapes in front of the parentheses:

`magick "$f" \( in/CorGold.png -gaussian-blur 0x10 \) -gravity center -composite -channel RGB +level-colors gold, out/CorGold.png`

**Reply by Rhinemann** [reply]
That's exactly it, thanks!
